build.bat

```diff
@@ -12,13 +12,20 @@ if exist C:\MSYS64\ (
 ) else (
     mkdir temp_install & cd temp_install
 
+    if exist C:\Windows\System32\curl (
```

Comment:
Why are you looking for the `curl` in a specific apathy and searching it in the `$PATH`?

Reply:
I was just adding a condition whether curl is present in his path.